tests_e2e/test_selenium.py

```diff
@@ -39,8 +39,8 @@ def delete_trello_board(board_id, api_key, token):
 @pytest.fixture(scope='module')
 def app_with_temp_board():
     # Loading environment variables 
-    file_path = find_dotenv('.env')
-    load_dotenv(file_path, override=True)
+    # file_path = find_dotenv('.env')
+    # load_dotenv(file_path, override=True)
```

Comment:
This shouldn't be necessary, I don't think this will throw an exception if `.env` doesn't exist, and removing it breaks `poetry run pytest` outside of Docker.
If it causes problems it's fine to take out though

Reply:
Thank you, I've enabled it. Let's see if it will break something again as I can't remember why I turned it off.